model/flow/role.go

```diff
@@ -110,20 +110,20 @@ func (r RoleList) Union(other RoleList) RoleList {
 }
 
 // Len returns length of the RoleList in the number of stored roles.
-// It satisfies the sort.Interface making the RoleList sortable.
+// It satisfies the sort.Type making the RoleList sortable.
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig